<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScatterWidget</class>
 <widget class="QWidget" name="ScatterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>3</horstretch>
    <verstretch>3</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>555</width>
    <height>500</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Scatter Plot</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QWidget" name="option_dashboard" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="xAxisLayout" stretch="0,4,0,0">
        <property name="spacing">
         <number>8</number>
        </property>
        <item>
         <widget class="QLabel" name="xlabel">
          <property name="text">
           <string>x axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="xAxisComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Set which attribute is plotted on the x axis</string>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToMinimumContentsLength</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="xLogCheckBox">
          <property name="toolTip">
           <string>Toggle on/off log scaling on the x axis</string>
          </property>
          <property name="text">
           <string>log</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="xFlipCheckBox">
          <property name="toolTip">
           <string>Flip/unflip the order of the x axis</string>
          </property>
          <property name="text">
           <string>flip</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="yAxisLayout" stretch="0,4,0,0">
        <property name="spacing">
         <number>8</number>
        </property>
        <item>
         <widget class="QLabel" name="ylabel">
          <property name="text">
           <string>y axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="yAxisComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Set which attribute is plotted on the y axis</string>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToMinimumContentsLength</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="yLogCheckBox">
          <property name="toolTip">
           <string>Toggle on/off log scaling on the y axis</string>
          </property>
          <property name="text">
           <string>log</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="yFlipCheckBox">
          <property name="toolTip">
           <string>Flip/unflip the order of the y axis</string>
          </property>
          <property name="text">
           <string>flip</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="snapLimits">
          <property name="toolTip">
           <string>Rescale plot limits to fit data</string>
          </property>
          <property name="text">
           <string>Auto scale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="swapAxes">
          <property name="toolTip">
           <string>Swap what's plotted on the x and y axes</string>
          </property>
          <property name="text">
           <string>Swap Axes</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="hidden_attributes">
        <property name="text">
         <string>show hidden attributes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>Plot Limits</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="xmin_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>x min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="xmin"/>
        </item>
        <item>
         <widget class="QLabel" name="xmax_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>x max</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="xmax"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="ymin_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>y min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ymin"/>
        </item>
        <item>
         <widget class="QLabel" name="ymax_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>y max</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ymax"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
